tests/mock-data.js

```diff
@@ -0,0 +1,47 @@
+import {
+  multipleSelectFilter,
+  selectFilter,
+} from '@/store/modules/queryFilters/filterModuleGenerators';
+import filterSubModule from '@/store/modules/queryFilters/filterSubModule';
+
+/**
+ * icecream is multiple select; you can have select 0 or more flavors
+ * @type {{mutations: {setValue}, state, getters: {toComponentProps, toSearchApi}}}
+ */
+const icecream = multipleSelectFilter({
```

Comment:
It seems to me that this behaviour is essentially already implemented by the Vuetify [`<select>`](https://vuetifyjs.com/en/components/selects/) component. They also have a `text` and a `value`, [`multiple`](https://vuetifyjs.com/en/api/v-select/#props-multiple) selects and through [`value`](https://vuetifyjs.com/en/api/v-select/#props-value) the default can be set.

Is there any particular reason why you chose to implement this yourself? Something I'm not seeing?

Reply:
This is a mock-module to test that data inserted into a module ends up properly in the components.